### PR TITLE
Add Linea Chain

### PR DESCRIPTION
**Pull request by nivcertora**
https://certora.atlassian.net/browse/CERT-7800

Ready for review but still waiting for @itay-certora proposal example for testing.

### CODE REVIEW

**Review by yoav-el-certora — CHANGES_REQUESTED** [review]
Lets put this on hold until @itay-certora will provide a proposal for regression testing

**Review by yoav-el-certora — APPROVED** [review]
Open a follow-up ticket to add Linea regression test, assign @itay-certora until he provides a proposal

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -13,8 +13,10 @@ jobs:
     if: |
       (
         (github.event_name == 'pull_request_review' && github.event.review.state == 'approved') ||
-        github.event.pull_request.user.login == 'nivcertora' ||
-        github.event.pull_request.user.login == 'liav-certora'
+        (github.event_name == 'pull_request' &&
```
Comment (liav-certora) [inline .github/workflows/CI.yml:16]: Adding another "Single Comment" to test CI trigger

**Review by liav-certora — DISMISSED** [review]

File: .github/workflows/CI.yml (hunk 1 above)
Comment (liav-certora) [reply .github/workflows/CI.yml:16]: Adding another "Single Comment" while PR is approved.

Comment (yoav-el-certora) [inline .github/workflows/CI.yml:16]:
```suggestion
        (github.event_name == 'pull_request_review' && github.event.review.state != 'commented') 
```

File: Quorum/tests/blockchain_api_test.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 
 @pytest.mark.parametrize(
-    "chain, contract_address",
+    'chain, contract_address',
```
Comment (yoav-el-certora) [inline Quorum/tests/blockchain_api_test.py:7]: Comment without approval

File: Quorum/apis/block_explorers/chains_api.py [hunk 3]
```diff
@@ -25,7 +25,8 @@ class ChainAPI:
         Chain.OPT: 10,
         Chain.POLY: 137,
         Chain.SCROLL: 534352,
-        Chain.ZK: 324
+        Chain.ZK: 324,
```
Comment (yoav-el-certora) [inline Quorum/apis/block_explorers/chains_api.py:28]: Comment

**Review by yoav-el-certora — APPROVED** [review]

**Pull request merged**